assignment/scripts/function-practice.js

```diff
@@ -50,40 +54,104 @@ console.log( 'isPositive - should say false', isPositive(-3) );
 // 6. Function to return the _last_ item in an array. If the 
 //    array is empty, return `undefined`.
 function getLast( array ) {
-
+  return array[array.length -1];
 }
 
+console.log('Should say undefined:', getLast([]));
+console.log('Should say 3:', getLast([1,2,3]));
+
 // 7. Function to find a value in an array. Return true if the 
 //    value is found and false otherwise. Use a loop;
 //    DO NOT use Array.includes, Array.indexOf, or Array.find 
 function find( value, array ){
-  
+  for(let i=0; i<array.length; i++){
+    if(array[i] === value){
+      return true;
+    }
+  }
+  return false;
 }
 
+console.log('Should say true:', find(3, [4, 5, 6, 4, 3, 1]))
+console.log('Should say false:', find(2, [1,4,5,6]))
+
 // ----------------------
 // Stretch Goals
 // ----------------------
 // 8. Function to check if a letter is the first letter in a 
 //    string. Return true if it is, and false otherwise
 function isFirstLetter(letter, string) {
-
+  let stringArray = string.split('');
+  if(letter === stringArray[0]){
```

Comment:
I love how you worked your way toward a solution based on what you knew with `split`.

That said, you don't actually need it. You can use `[0]` on a string directly.